pkgs/by-name/co/cosmic-session/package.nix

```
@@ -8,19 +8,20 @@
   stdenv,
   xdg-desktop-portal-cosmic,
 }:
-rustPlatform.buildRustPackage rec {
+
+rustPlatform.buildRustPackage (oldAttrs: {
```

```suggestion
rustPlatform.buildRustPackage (finalAttrs: {
```
`finalAttrs` naming should be used instead. `oldAttrs` is used for `overrideAttrs`